### ISSUE TITLE
Bug  when using the app

### ISSUE THREAD

**Issue by seeyoga** [nextcloud/android#7579]
### Steps to reproduce
1. 
2. 
3. 

### Expected behaviour
- Tell us what should happen

### Actual behaviour
- Tell us what happens

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by seeyoga** [comment]
Logfile from mobile application 

************ CAUSE OF ERROR ************

android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{e7dae1b u0 com.nextcloud.client/.media.PlayerService}
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1872)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7050)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30140090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: samsung
Device: dreamlte
Model: SM-G950F
Id: PPR1.180610.011
Product: dreamltexx

************ FIRMWARE ************
SDK: 28
Release: 9
Incremental: G950FXXSBDTJ1

**Comment by tobiasKaminsky** [comment]
This will be fixed via 3.14.1 coming today.
